sycl/source/detail/program_impl.cpp

```diff
@@ -26,6 +27,21 @@ __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 namespace detail {
 
+inline const std::string GetBackendString(cl::sycl::backend backend) {
```

Comment:
if you really wanted to name the backend in the error message (I don't feel this is necessary) then you'd employ the existing https://github.com/intel/llvm/blob/cc3d0956c46135b2791d9adaf191fe82ff67eb8e/sycl/include/CL/sycl/backend_types.hpp#L31

Reply:
I'll skip this if not necessary.